src/js/rx/reducers/alert.js

```diff
@@ -9,7 +10,7 @@ const initialState = {
 export default function alert(state = initialState, action) {
   switch (action.type) {
     case 'OPEN_ALERT':
-      return Object.assign({}, state, {
+      return assign(state, {
```

Comment:
Since we're overwriting everything in the state here, I guess we don't really need `assign` at all, and can just return the object directly as `{ content: action.content, status:action.status, visible: true }`.

And for that matter, closing the alert can just return the `initialState`.